### DISCUSSION TITLE
How do I start this project locally?

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/coder#8313] [General] [answered]
How do I start this project locally?
Hello, my friend. I want to start this open source project locally, so that I can learn the source code more deeply, how to start it, do you know the document?

**Comment by matifali** [comment, marked as answer]
Easiest will be to use [`nix-shell`](https://coder.com/docs/v2/latest/CONTRIBUTING#development-workflow)

**Comment by matifali** [comment]
we also have a [`.decontainer`](https://github.com/coder/coder/tree/main/.devcontainer) if you prefer that.
